### ISSUE TITLE
code problem

### ISSUE THREAD

**Issue by whyang78** [liuyuemaicha/simple_faster_rcnn#1]
`  
rois = torch.from_numpy(sample_roi).float()  
roi_indices = 0 * np.ones((len(rois),), dtype=np.int32)  
roi_indices = torch.from_numpy(roi_indices).float()  
print(rois.shape, roi_indices.shape)  
将图像的索引号和预测的有效框进行合并, 这样我们将会得到维度是[N, 5]  5=>(index, x1, y1, x2, y2)的张量    
indices_and_rois = torch.cat([roi_indices[:, None], rois], dim=1)
`  
rois维度是[index,y1,x1,y2,x2]，所以后面的roi pooling那里代码也有问题。可以在最后添加  
`xy_indices_and_rois = indices_and_rois[:, [0, 2, 1, 4, 3]]
indices_and_rois = xy_indices_and_rois.contiguous()`
